test/liquidity-protection/LiquidityProtection.js

```diff
@@ -2920,94 +3133,6 @@ describe('LiquidityProtection', () => {
                         await setTime(BigNumber.from(1));
                     });
 
-                    for (const config of CONFIGURATIONS) {
-                        for (const numOfDays of NUM_OF_DAYS) {
-                            const timestamp = numOfDays * 24 * 60 * 60 + 1;
-                            for (const decimals of DECIMAL_COMBINATIONS) {
-                                const amounts = decimals.map((n) => BigNumber.from(10).pow(BigNumber.from(n)));
-
-                                let test;
-                                if (!config.increaseRate && !config.generateFee) {
-                                    test = (actual, expected) =>
-                                        condOrAlmostEqual(
-                                            actual.eq(expected),
-                                            actual,
-                                            expected,
-                                            { 1: '0.000000000000001', 3: '0.00000004' }[converterType]
-                                        );
-                                } else if (!config.increaseRate && config.generateFee) {
-                                    test = (actual, expected) =>
-                                        condOrAlmostEqual(
-                                            actual.gt(expected),
-                                            actual,
-                                            expected,
-                                            { 1: '0.0', 3: '0.0' }[converterType]
-                                        );
-                                } else if (config.increaseRate && !config.generateFee && numOfDays < 100) {
-                                    test = (actual, expected) =>
-                                        condOrAlmostEqual(
-                                            actual.lt(expected),
-                                            actual,
-                                            expected,
-                                            { 1: '0.0', 3: '0.0' }[converterType]
-                                        );
-                                } else if (config.increaseRate && !config.generateFee && numOfDays >= 100) {
-                                    test = (actual, expected) =>
-                                        condOrAlmostEqual(
-                                            actual.eq(expected),
```

Comment:
I wanna reinstate this entire block, and just change `eq` to `lt` here (which will that when the rate has changed AND no fees were generated AND the vesting period has ended, the user is bound to lose). The other edge cases remain unchanged, and will serve as part of a more thorough test suite.

Reply:
go for it